generator/pom.xml

```diff
@@ -180,6 +180,23 @@
                     </configuration>
                 </plugin>
 
+                <!-- Generate UML diagrams in javadoc using doclet: mvn javadoc:javadoc -->
```

Comment:
This was added for the UML diagrams but these are no longer in the PR? Probably not a big deal

Reply:
This was added so that UML diagrams are produced within the javadoc, not for the separate design diagrams.

Reply:
ah ok 👍